### PR TITLE
New Content: Community Manager Self-Care Chapter 

### PR DESCRIPTION
**Pull request by samus-aran**
This PR contains the main bulk of the the Community Manager Self-Care chapter. The original outline can be found [here](https://github.com/theopensourceway/guidebook/blob/87a08f14e0d63677685c237b5bc30aef55cb23d3/community-manager-self-care.adoc) and a more detailed one [here](https://github.com/theopensourceway/guidebook/pull/69).

I have a few comments to make:

- I assume we all agree to use *American British* spelling. Not a big issue, just need to ensure editors from different regions are aware of that.
- This became an extremely large chapter (from my brief comparison to others). The subject was the community manager themselves and highlighting to them the importance of self-care from their perspective. The spill out from their 'enlightenment' and actions would then cascade to the team and community. I feel that there could be some expansion on this due to the [discussions from the chapter's GitHub issue](https://github.com/theopensourceway/guidebook/issues/70#issuecomment-611615120) but I'm not sure how and where to place it... as I said, this chapter is big and perhaps should be relocated outside of the "Avoid pitfalls" category since we want people to think about this as the foundation of building their communities rather than an afterthought. My only suggestion is perhaps have an umbrella category of "Dark Side of Communities" or "Emotional Side to Communities" which allows authors to add more topics like *trolling*, *resolving conflict*, or *bereavements and deceased user policies* etc. not a biggie at the moment, just considering how large the chapter will be once we get some interviews and personal stories from others in the industry, where they sit.
- Considering putting in a conclusion to try and pull together the chapter.. added to the to-do list
- Structure of adding in resources. I have added a few footnotes but what would be the best for resources? Simply a chapter heading at the bottom?
- Also a consideration, a reaffirmation that this is not medical advice at the end of the chapter... Yes we have it at the top and a couple of times highlighted that those feel they require immediate help to do so.. just a thought.
- Adding in a few suggestions of SME's but would love to hear about more :)

*SME suggestions*
- Mary Thengvall (@mary-grace)
- Anjuan Simmons and Dr Aneika Simmons (Managing the Burnout Burndown - https://newyork2019.theleaddeveloper.com/talks )
- Would be great to source medical practitioners... 
- If you have any more suggestions a message in the comments would be gooooood.

I have a few ideas on the interviewees as I feel this chapter would need personal reflection to really bring this topic "home". Again we need to ask the question on how best to ask volunteers to participate in writing down personal and perhaps traumatic experiences. There is a lot that should be handle with care and respect especially around this topic. I think I'll take that to the mailing list for further discussions as soon as I can.

- [x] Add Resources to the chapter
- [ ] Add a Conclusion?
- [ ] Any additional formatting, bold or italics
- [ ] Highlight where it would be good to add references to resources or "personal" stories within the chapter
- [x] Create discussion on mailing list on Interviewees (or SME's too)...

### CODE REVIEW

**Comment by vmbrasseur** [comment]
Burnout book from a medical professional: https://www.theburnoutproject.com.au/product/burnoutbookpaperback/

**Comment by samus-aran** [comment]
> Burnout book from a medical professional: https://www.theburnoutproject.com.au/product/burnoutbookpaperback/

Super, thanks!

**Comment by samus-aran** [comment]
Note to self here: When writing content up as extensive as this to continuously keep track of references whilst writing.. 19f60e4 is the first commit, as I still have other references to cite. It's a pain to do it retrospectively and tracing through my browser history :sob: 
Also a consideration for the editorial team about plagiarism. I use Quetext paid (relatively cheap) program but don't know what process the team would need to do and who's responsibility it is.

**Comment by samus-aran** [comment]
Hey everyone, I feel I'm holding this chapter back for not real reason. So I will just shout out to @semioticrobotic @shaunix or @bproffitt if they could have a quick review of this chapter so it can get be 'officially' put into the editorial process.

Just on a note I have and will continue reach out to SMEs from local Universities in medical and psychology field.

File: community-manager-self-care.adoc
Comment (semioticrobotic) [inline community-manager-self-care.adoc:28]: I feel like this is a nice opportunity to provide readers with a cursory preview or roadmap of the chapter ahead.

File: community-manager-self-care.adoc
Comment (semioticrobotic) [inline community-manager-self-care.adoc:109]: This sentence reads a bit awkwardly to me, but I can't quite determine the point you're making, so I'm unsure how best to edit it. Can you help?

File: community-manager-self-care.adoc
Comment (semioticrobotic) [inline community-manager-self-care.adoc:117]: Similar comment to line 109: I think the sentence could use some editing, but I'm not quite sure of your intent with it, so I'm not sure how best to help.

File: community-manager-self-care.adoc
Comment (semioticrobotic) [inline community-manager-self-care.adoc:83]: My only concern about this section in general is that it advances some claims that could use supporting evidence (or at least additional reading!), perhaps available upon review from another subject-matter expert.

File: community-manager-self-care.adoc
Comment (samus-aran) [reply community-manager-self-care.adoc]: Yup, I've updated this. :+1:

File: community-manager-self-care.adoc
Comment (samus-aran) [reply community-manager-self-care.adoc:83]: Understandable, as I said in my PR that retrospectively pulling in my references I bet I have missed some. I can't cite some of the points I believe you are highlighting but I can see the ones I can, so I'll schedule time to check that and update!

But yes, it's good to highlight these so that the SME can provide their input :+1:

File: community-manager-self-care.adoc
Comment (semioticrobotic) [reply community-manager-self-care.adoc]: Awesome! Thanks.

File: community-manager-self-care.adoc
Comment (semioticrobotic) [reply community-manager-self-care.adoc:83]: All makes sense to me and I'm in agreement. Thanks for considering.

File: community-manager-self-care.adoc
Comment (samus-aran) [reply community-manager-self-care.adoc]: I have made amendments to hopefully make it a bit more readable. Let me know if it requires further clarification!

File: community-manager-self-care.adoc
Comment (samus-aran) [reply community-manager-self-care.adoc]: (similarly to the other review) I have made amendments to hopefully make it a bit more readable. Let me know if it requires further clarification!

File: community-manager-self-care.adoc
Comment (semioticrobotic) [reply community-manager-self-care.adoc]: It does help, and I will perhaps offer another tweak or two, now that I see the direction. Thanks.

File: community-manager-self-care.adoc
Comment (semioticrobotic) [reply community-manager-self-care.adoc]: Great! Thanks.

File: community-manager-self-care.adoc
Comment (samus-aran) [reply community-manager-self-care.adoc:83]: Okay dokay, I've added the remaining citations I have. When the SME is on board, this section will be highlighted requiring additional references to help verify the content. :+1:

**Comment by samus-aran** [comment]
Ta-da :tada: 

So this should be ready to be have it's initial "quick review" before merged into main (let me know if there is anything obvious).

I have also included *Status: draft* for future prep which we can update if a better solution follows.

Next step (once this is merged) is to move the issue into 'Subject Matter Review' on the board. :+1:

**Comment by samus-aran** [comment]
Okay dokay. I'm not going to merge this in just in case the process had changed since a couple of weeks ago. But the moment after tomorrows weekly I'll hit the merge button :+1:

**Review by bproffitt — APPROVED** [review]
Based on draft review criteria, this chapter is ready to merge.

**Pull request merged**
